mandatory/2-function-creation.js

```diff
@@ -4,7 +4,13 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+function isAcceptableUser(userAge, isLoggedIn) {
+  if (userAge >= 18 && isLoggedIn === true) {
```

Comment:
Should we use true two times? It is a qustion.

Reply:
This === compare if value is true for example if 2 === 2